### ISSUE TITLE
Error when running on build

### ISSUE THREAD

**Issue by andreGoncalvesNyra** [ammariqais/SkiaForUnity#54]
Hello 👋 

I was trying to use `SkiaForUnity`with `SkottiePlayerV2`
It works fine on editor, but when I build it targeting mac desktop I get the following error:

```
Exception: DllNotFoundException: libSkiaSharp assembly:<unknown assembly> type:<unknown type> member:(null)


SkiaSharp.SKData..cctor () (at ./Library/PackageCache/com.ammariqais.skiaforunity@18b9511a3e/SkiaSharp-Bindings/SKData.cs:22)
Rethrow as TypeInitializationException: The type initializer for 'SkiaSharp.SKData' threw an exception.
SkiaSharp.Unity.SkottiePlayerV2.LoadAnimation (System.String json) (at ./Library/PackageCache/com.ammariqais.skiaforunity@18b9511a3e/Runtime/SkottiePlayerV2.cs:119)
SkiaSharp.Unity.SkottiePlayerV2.Start () (at ./Library/PackageCache/com.ammariqais.skiaforunity@18b9511a3e/Runtime/SkottiePlayerV2.cs:68)
```

How can I fix it?

**Comment by Phlegmati** [comment]
*Bump*

As I am running into the same problem. Is there any assembly definition reference or player configuration I am missing? 
(I am using .NET Framework, btw.)

**Comment by ammariqais** [comment]
@Phlegmati fixed on version 3.1.3 please update the package
